packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -1,57 +1,110 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-meter-background-color: Specifies the background color of the component.
+ * @prop --calcite-meter-border-color: Specifies the border color of the component and displayed step lines.
+ * @prop --calcite-meter-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-meter-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-meter-fill-color: Specifies the color of the component's fill.
+ * @prop --calcite-meter-range-text-color: Specifies the color of the component's range labels.
+ * @prop --calcite-meter-value-text-color: Specifies the color of the component's value label.
+ *
+ */
+
+/**
+  * Local props
+  * These properties are intended for internal component use only. It is not recommended that these properties be overwritten.
+  *
+  * --calcite-internal-meter-fill-color
+  * --calcite-internal-meter-background-color
+  * --calcite-internal-meter-space
+  * --calcite-internal-meter-height
+  * --calcite-internal-meter-font-size
+*/
+
 @include base-component();
 @include disabled();
 
 :host {
   @apply flex;
-  --calcite-meter-space-internal: theme("spacing[0.5]");
-  --calcite-meter-height-internal: theme("spacing.4");
-  --calcite-meter-font-size-internal: var(--calcite-font-size--1);
+  --calcite-internal-meter-space: var(--calcite-spacing-base);
+  --calcite-internal-meter-height: var(--calcite-spacing-lg);
+  --calcite-internal-meter-font-size: var(--calcite-font-size--1);
+  --calcite-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-brand));
+  --calcite-meter-background-color: var(--calcite-color-foreground-2);
+  --calcite-meter-border-color: var(--calcite-color-border-3);
+  --calcite-meter-shadow: var(--calcite-shadow-none);
+  --calcite-meter-range-text-color: var(--calcite-color-text-3);
+  --calcite-meter-value-text-color: var(--calcite-color-text-1);
+  --calcite-meter-shadow: var(--calcite-shadow-none);
```

Comment:
This line's a duplicate of [L38](https://github.com/Esri/calcite-design-system/pull/11757/files#diff-77aafbee074640b5cd6983b9c311ccb7e5af1410fb3d5b27d8610513dfd4b620R38).